src/toolbar_editor.cpp

```diff
@@ -28,10 +28,13 @@ Q_DECLARE_METATYPE(QMenu*)
 Q_DECLARE_METATYPE(QToolBar*)
 Q_DECLARE_METATYPE(QAction*)
 
+const QString Toolbar_Editor::customToolbarNamePrefix = QString("custom_toolbar_");
```

Comment:
I don't see where my code treats custom toolbars differently, what I don't like is checking their name when you determine whether to remove it or not.